### ISSUE TITLE
Need regular navigation menu with sub menu items

### ISSUE THREAD

**Issue by simplyaa** [Simplero/Roadmap#357]
We really need a regular menu system that allows you to have sub-menu items For example under Work With Me menu item we need to be able to have pages under that on the menu (not just in the sidebar). This is crucial to have.

**Comment by calvincorreli** [comment]
There's a theme editor. You can always hire someone with basic HTML/CSS/JS to code it up for you however you want it. Go to Theme and select Customize code from the drop-down.

**Comment by simplyaa** [comment]
I do know basic HTML and CSS but not Javascript. I see it uses Liquid that I've never worked with before. I found an article on Shopify that says to add this that looks like it may work:

{% assign menu = linklists.main-menu %}
  <pre>
  {{ menu.title }}: {{ menu.levels }}
  {% for link in menu.links %}
    {{ link.title }}: {{ link.levels }}
    {% for sub_link in link.links %}
      {{ sub_link.title }}: {{ sub_link.levels }}
      {% for sub_sub_link in sub_link.links %}
        {{ sub_sub_link.title }}: {{ sub_sub_link.levels }}
      {% endfor %}
    {% endfor %}
  {% endfor %}
  </pre>

Is this the code we need? I'm trying to get this resolved for 2 different Simplero accounts for myself and my client.

**Comment by Livskvalitetspartner** [comment]
I totally agree. I dont use membership sites yet cause there is no sub menu. I was astonnished you did not make it yet. So please .... make it possible to make a good menu. I need a menu for Problems i can solve: And under that i have 60-100 different things, i need in a menu.  If you dont change the menu ill have to find some other solution and id hate that because i love simplero, in so many other ways. And as i understand you stopped making adjustments in spaces with i used so far.

**Comment by emilyannpeterson** [comment]
Plus 1 for a drop-down menu option in membership sites!

**Comment by teresa-slp** [comment]
I agree with Livskvalitetspartner.

I've put off using the Simplero member site for two years because I cannot make it look and function like my current site. I've given up on the visual part, but really need to have a better looking and easier functioning navigation method than the side bar.

I have to get away from Optimize Press which is my theme and my member site. It has too many coding issues and customer service is poor. Therefore, I am giving in and using the Simplero member site because of the awesome customer support, payment and clever email list options.

However, the top drop down menu really should be a staple. Trying to create a large course using the side bar as navigation is a nightmare!

Thanks for all your help and steady flow of updates and improvements, Calvin and team!


Below is a pic of my current member site. 
<img width="1352" alt="drop-down nav for courses" src="https://user-images.githubusercontent.com/31087371/52444501-6b6dae80-2add-11e9-99d9-507ce90e4e46.png">
